lib/resources/r4/service_request.yaml

```diff
@@ -0,0 +1,94 @@
+---
+  name: ServiceRequest
```

Comment:
I don't think we need to get terminology bindings for all the fields. Lets go by what we have recorded on OPENSVC-13513

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/389/commits/864148c6029d87dff09205f010eaf24ddb7a284e